bowling-ball/Game.cs

```diff
@@ -4,16 +4,56 @@ namespace BowlingBall
 {
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
+        int[] Rolls = new int[21];
+
+        public void Roll(int[] pins)
+        {
+            int i=0;
+            foreach(int iterator in pins)
+            {
+                Rolls[i]=iterator;
+                i=i+1;
+            }
         }
 
-        public int GetScore()
+        public int GetScore()
         {
-            throw new NotImplementedException();
+            int score = 0;
+            int FrameIndex = 0;
+            for (int i = 0; i < 10; i++) {
+                if (IsStrike(FrameIndex)) {
+                    score = score + 10 + StrikeBonus(FrameIndex);
+                    FrameIndex = FrameIndex + 1;
+                } else if (IsSpare(FrameIndex)) {
+                    score =score + 10 + SpareBonus(FrameIndex);
+                    FrameIndex = FrameIndex + 2;

```

Comment:
These is a special logic and needs to be documented why we applied this logic,
ex. - we are considering 2 pins (1 frame) and hence increased index by 2.